react/components/ExtensionPoint.tsx

```diff
@@ -191,22 +191,16 @@ const ExtensionPoint: FC<Props> = props => {
     </ExtensionPointComponent>
   )
 
-  if (runtime.preview && isRootTreePath) {
-    return (
-      <Fragment>
-        <LoadingBar />
-        {extensionPointComponent}
-      </Fragment>
-    )
-  }
-
   // "client" component assets are sent to server side rendering,
   // but they should display a loading animation.
   //
   // "lazy" components might never be used, so they don't necessarily
   // need a loading animation.
   return renderStrategy === 'client' && !runtime.amp ? (
-    <NoSSR onSSR={<Loading />}>{extensionPointComponent}</NoSSR>
+    <NoSSR onSSR={<Loading />}>
+      {runtime.preview && !runtime.amp && <LoadingBar />}
```

Comment:
this `!runtime.amp` is not needed here, we already know its true based on the condition above

Reply:
Wrong verification, the correct is `runtime.preview && isRootTreePath && <LoadingBar />`

Reply:
the `isRootTreePath` variable is not used anymore, only @lbebber can tell us what should happen